pkg/controller/plan/adapter/vsphere/builder_test.go

```diff
@@ -17,30 +17,35 @@ import (
 var builderLog = logging.WithName("vsphere-builder-test")
 
 const ManualOrigin = string(types.NetIpConfigInfoIpAddressOriginManual)
+const TestGuestNetworkMac = "00:50:56:83:25:47"
+const TestGuestNetworkIp = "172.29.3.193"
+const ExpectedMacStaticIps = "00:50:56:83:25:47:ip:172.29.3.193"
+
+var TestGuestNetwork = vsphere.GuestNetwork{MAC: TestGuestNetworkMac, IP: TestGuestNetworkIp, Origin: ManualOrigin}
 
 var _ = Describe("vSphere builder", func() {
 	builder := createBuilder()
 	DescribeTable("should", func(vm *model.VM, outputMap string) {
 		Expect(builder.mapMacStaticIps(vm)).Should(Equal(outputMap))
 	},
 		Entry("no static ips", &model.VM{GuestID: "windows9Guest"}, ""),
-		Entry("single static ip", &model.VM{GuestID: "windows9Guest", GuestNetworks: []vsphere.GuestNetwork{{MAC: "00:50:56:83:25:47", IP: "172.29.3.193", Origin: ManualOrigin}}}, "00:50:56:83:25:47:ip:172.29.3.193"),
+		Entry("single static ip", &model.VM{GuestID: "windows9Guest", GuestNetworks: []vsphere.GuestNetwork{TestGuestNetwork}}, ExpectedMacStaticIps),
 		Entry("multiple static ips", &model.VM{GuestID: "windows9Guest", GuestNetworks: []vsphere.GuestNetwork{
 			{
-				MAC:    "00:50:56:83:25:47",
-				IP:     "172.29.3.193",
+				MAC:    TestGuestNetworkMac,
+				IP:     TestGuestNetworkIp,
 				Origin: ManualOrigin,
 			},
 			{
-				MAC:    "00:50:56:83:25:47",
+				MAC:    TestGuestNetworkMac,
 				IP:     "fe80::5da:b7a5:e0a2:a097",
 				Origin: ManualOrigin,
 			},
 		},
 		}, "00:50:56:83:25:47:ip:172.29.3.193_00:50:56:83:25:47:ip:fe80::5da:b7a5:e0a2:a097"),
-		Entry("non-static ip", &model.VM{GuestID: "windows9Guest", GuestNetworks: []vsphere.GuestNetwork{{MAC: "00:50:56:83:25:47", IP: "172.29.3.193", Origin: string(types.NetIpConfigInfoIpAddressOriginDhcp)}}}, ""),
-		Entry("non windows vm", &model.VM{GuestID: "other", GuestNetworks: []vsphere.GuestNetwork{{MAC: "00:50:56:83:25:47", IP: "172.29.3.193", Origin: ManualOrigin}}}, ""),
-		Entry("no OS vm", &model.VM{GuestNetworks: []vsphere.GuestNetwork{{MAC: "00:50:56:83:25:47", IP: "172.29.3.193", Origin: ManualOrigin}}}, ""),
+		Entry("non-static ip", &model.VM{GuestID: "windows9Guest", GuestNetworks: []vsphere.GuestNetwork{{MAC: TestGuestNetworkMac, IP: TestGuestNetworkIp, Origin: string(types.NetIpConfigInfoIpAddressOriginDhcp)}}}, ""),
+		Entry("non windows vm", &model.VM{GuestID: "other", GuestNetworks: []vsphere.GuestNetwork{TestGuestNetwork}}, ExpectedMacStaticIps),
+		Entry("no OS vm", &model.VM{GuestNetworks: []vsphere.GuestNetwork{TestGuestNetwork}}, ExpectedMacStaticIps),
```

Comment:
well, if we have no OS, then we won't have any details of the vNIC GuestNetworks

Reply:
do you suggest to remove the "no OS vm" test ?

Reply:
> I would put into the commit that on a following patch you are going to add virt-customize script to support RHEL. That will add some value of this PR as a preparation step.

Added to the PR description

Reply:
and the commit message